### ISSUE TITLE
1.13.0 doesn't work with CocoaPods

### ISSUE THREAD

**Issue by Kapeli** [sparkle-project/Sparkle#713]
I can't get Sparkle 1.13.0 to work when installed with CocoaPods.

I get this error:

```
dyld: Library not loaded: @rpath/Sparkle.framework/Versions/A/Sparkle
  Referenced from: /Users/bogdan/Library/Developer/Xcode/DerivedData/Sparkle_CocoaPods_Test-axaeaymvwyoegkhehobcwvgpvftn/Build/Products/Debug/Sparkle_CocoaPods_Test.app/Contents/MacOS/Sparkle_CocoaPods_Test
  Reason: image not found
```

Sample Xcode project:

[Sparkle_CocoaPods_Test.zip](https://github.com/sparkle-project/Sparkle/files/84004/Sparkle_CocoaPods_Test.zip)

Using Sparkle 1.11 instead of 1.13 works.

**Comment by pornel** [comment]
In Build Settings tab set “Runpath Search Paths” to `@loader_path/../Frameworks`

**Comment by pornel** [comment]
We do have 

```ruby
  s.xcconfig            = {
    'FRAMEWORK_SEARCH_PATHS' => '"${PODS_ROOT}/Sparkle"',
    'LD_RUNPATH_SEARCH_PATHS' => '@loader_path/../Frameworks'
  }
```
in the podfile, so I don't know why it's not working for you.

**Comment by Kapeli** [comment]
Maybe because I'm using CocoaPods 1.0.0.beta.2?

**Comment by Kapeli** [comment]
I've tried setting the "Runpath Search Path" to `@loader_path/../Frameworks` and it doesn't fix it in the sample project.

Also if I check the search path for the Pods target it's set to `@loader_path/../Frameworks` so CocoaPods seems to set it as well.

![screen shot 2016-01-10 at 16 18 17](https://cloud.githubusercontent.com/assets/1445635/12221965/e6e3397a-b7b5-11e5-9797-c53109adb17c.png)

**Comment by jakepetroules** [comment]
Does `/Users/bogdan/Library/Developer/Xcode/DerivedData/Sparkle_CocoaPods_Test-axaeaymvwyoegkhehobcwvgpvftn/Build/Products/Debug/Sparkle_CocoaPods_Test.app/Contents/Frameworks/Sparkle.framework` actually exist?

**Comment by Kapeli** [comment]
Nope. `Sparkle.framework` is nowhere inside the app bundle.

**Comment by jakepetroules** [comment]
Then that's your problem. It needs to be embedded into your app bundle, of course. Add it in your target's "Embedded Frameworks" section.

**Comment by Kapeli** [comment]
It works after I do that. Any idea why I have to do this manually for Sparkle 1.13.0? For Sparkle 1.11.1 CocoaPods seems to handle it and I don't have to do it.

**Comment by cmsj** [comment]
I'm seeing the same thing after updating CocoaPods to include 1.13.0 in [Hammerspoon](https://github.com/Hammerspoon/hammerspoon)

It works if I manually add `Pods/Sparkle/Sparkle.framework` as an Embedded Framework, but I've never needed to do that before, and I only want Sparkle included in release builds (which is specified in my Podfile).

**Comment by cmsj** [comment]
Actually, belay that, I realised I was on Cocoapods 0.38.2. 0.39 includes a fix for embedding dynamic frameworks in app bundles (and fwiw, one may have to do "pod install" after upgrading, to make it rewrite all the right things)

**Comment by xhacker** [comment]
I think it’s a CocoaPods issue. I tried running `pod install` in your sample project with CocoaPods 0.39.0 and it worked. Looks like the following code is missing in your `Pods-Sparkle_CocoaPods_Test-frameworks.sh` so that the framework won’t be copied.

```sh
if [[ "$CONFIGURATION" == "Debug" ]]; then
  install_framework "${PODS_ROOT}/Sparkle/Sparkle.framework"
fi
if [[ "$CONFIGURATION" == "Release" ]]; then
  install_framework "${PODS_ROOT}/Sparkle/Sparkle.framework"
fi
```

**Comment by d-metzler** [comment]
I have CocoaPods 0.39.0 and our project was working with the Sparkle pod for 1.11.1.  With 1.13 the pod breaks and the Sparkle.framework is not copied to the app bundle.

This seems to be a pod issue with Sparkle.  Any plans on a fix for this?

**Comment by jakepetroules** [comment]
The only difference in the podspec between 1.11.1 and 1.13 is the version number. Also, you should upgrade CocoaPods to the latest version. Lastly, "the pod breaks" doesn't really tell us anything so please give more details on exactly what happens.

We should have some tests for CocoaPods in CI; I'm sure this is feasible.

**Comment by pornel** [comment]
Please reopen if it still doesn't work with 1.15.

**Comment by ohwutup** [comment]
Did anyone have a fix for using 1.11.1 via CocoaPods?

**Comment by pornel** [comment]
1.11 contains security vulnerability. Please don't use it.
